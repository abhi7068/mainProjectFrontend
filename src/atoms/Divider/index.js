import React from 'react';

import { makeStyles } from '@material-ui/core/styles';
import PropTypes from 'prop-types';
const useStyles = makeStyles((theme) => ({
  divider: {
    color: '#878787',
    margin: '16px 0',
    textAlign: 'center',
  },
}));

function Divider({ children }) {
  const classes = useStyles();
  return <div className={classes.divider}>{children}</div>;
}
Divider.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
};
export default Divider;
