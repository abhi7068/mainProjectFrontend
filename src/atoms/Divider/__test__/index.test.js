import React from 'react';
import Divider from '../index';
import { render, cleanup } from '@testing-library/react';
afterEach(cleanup);
it('it renders', () => {
  render(<Divider />);
});
it('snapshot', () => {
  const { asFragment } = render(<Divider />);
  expect(asFragment()).toMatchSnapshot();
});
