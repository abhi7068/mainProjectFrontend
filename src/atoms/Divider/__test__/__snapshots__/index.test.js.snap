// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-divider-2"
  />
</DocumentFragment>
`;
